Formula/i/ip.rb

```diff
@@ -0,0 +1,37 @@
+class Ip < Formula
+  desc "Display public and private IPs and city for active network interfaces"
+  homepage "https://github.com/StarkChristmas/ipget"
+  url "https://github.com/StarkChristmas/ipget/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "0f1e6099eb568f3027db981177a1062c84036171cb13b697c38c992a81339ba0"
+  license "MIT"
+  head "https://github.com/StarkChristmas/ipget.git", branch: "main"
+
+  depends_on "go" => [:build, ">= 1.18"]
```

Comment:
This is invalid syntax, use
```suggestion
  depends_on "go" => :build
```